packages/core/useWebSocket/index.ts

```diff
@@ -223,7 +223,7 @@ export function useWebSocket<Data = any>(
     heartbeatResume = resume
   }
 
-  if (options.immediate) _init()
+  if (options.immediate === false) _init()
```

Comment:
emm?

Reply:
Oops, seems it was very late yesterday 🤣
Will change it when I'm out of bed